mev_inspect/classifiers/specs/compound.py

```diff
@@ -1,16 +1,76 @@
+from typing import List, Optional
+
 from mev_inspect.schemas.classifiers import (
+    Classification,
+    ClassifiedTrace,
     ClassifierSpec,
+    DecodedCallTrace,
     LiquidationClassifier,
     SeizeClassifier,
 )
+from mev_inspect.schemas.liquidations import Liquidation
 from mev_inspect.schemas.traces import Protocol
+from mev_inspect.schemas.transfers import Transfer
+
+
+class CompoundLiquidationClassifier(LiquidationClassifier):
+    @staticmethod
+    def parse_liquidation(
+        liquidation_trace: DecodedCallTrace,
+        child_transfers: List[Transfer],
+        child_traces: List[ClassifiedTrace],
+    ) -> Optional[Liquidation]:
+
+        liquidator = liquidation_trace.from_address
+        liquidated = liquidation_trace.inputs["borrower"]
+        debt_token_address = liquidation_trace.to_address
+        debt_purchase_amount = liquidation_trace.value
+        received_token_address = liquidation_trace.inputs["cTokenCollateral"]
+        received_amount = 0
```

Comment:
if 0 is a default meaning "absent" use `None` instead

Reply:
Great call